### PR TITLE
Update only development

### PR DESCRIPTION
**Pull request by enarha**
The changes generated by this pipelinerun are used to run the Konflux CI and that uses only `components/pipeline-service/development`. Removing the update of `components/pipeline-service/staging` and `components/monitoring/grafana/base/dashboards/pipeline-service` because it is not actually used and just adds "noise" to the changes.

### CODE REVIEW

**Review by gabemontero — APPROVED** [review]
good catch / LGTM

**Comment by gabemontero** [comment]
feel free to merge at your discretion @enarha

**Comment by enarha** [comment]
@gabemontero I'll merge after #1010 1010 is merged, so we can verify `yq` exists on the image we use. IMO just merge this after #1010 if I'm not around.

**Pull request closed without merging**
